### DISCUSSION TITLE
Questions about nsisconfig

### DISCUSSION THREAD

**Discussion by EHW-liao** [tauri-apps/tauri#7034] [Q&A] [answered]
I successfully generated the exe installation package using tauri1.3. Now I want to add my own NSH configuration file during the packaging process, similar to the wxs configuration in wixconfig. How can I implement it? I checked the nsisconfig on the official website and did not mention this configuration.

**Comment by FabianLars** [comment, marked as answer]
v1.4 will have an option to change the template: https://github.com/tauri-apps/tauri/pull/6922. If you don't want to wait you could use the cli from git: https://tauri.app/v1/guides/faq#how-can-i-use-unpublished-tauri-changes

**Reply by EHW-liao** [reply]
thank you.May I ask another question? Can the embedBootstrapper in WebviewInstallMode specify a URL address for download? I would like to download the webView2 installation package from the URL address we specified during user installation
